### ISSUE TITLE
One for the FAQ: what does DP-3T add over the Google/Apple API

### ISSUE THREAD

**Issue by danbogdanov** [DP-3T/documents#240]
I've been hearing this question from several sources so it might be a Frequently Asked Question.

What is the added value of DP-3T over just using the Apple/Google API in an app? E.g., after May, should a country just build an app around the Google/Apple API and sidestep DP-3T?

Thanks!

**Comment by Vanuan** [comment]
First of all, it's open source. You won't be able to inspect A/G implementation without some reverse engineering.

**Comment by danbogdanov** [comment]
> Thirdly, DP-3T might adopt Google/Apple API and provide a more high-level API on top of it.

The question here was, should a government implement the A/G API straight, keeping the app simpler? Which functions will DP-3T provide on top of A/G API to support the adopters?

**Comment by lbarman** [comment]
Hi all, my answer is to be taken with a grain of salt; I'm also somewhat speculating. With the info I have, I'd say that A/G will provide the API for Bluetooth contact tracing itself, but nothing about the backend / the authorization with health services / the app itself / interoperability between regions etc.

**Comment by Vanuan** [comment]
@danbogdanov you should do your own research.

Here's a set of APIs provided by Android:
https://www.blog.google/documents/61/Android_Exposure_Notification_API_v1.1.pdf

Here's a set of APIs provided by DP3T:
https://github.com/DP-3T/dp3t-sdk-android/blob/develop/README.md#methods

It's true that there's a lot of overlap. Here are the methods you have to implement on top of Android API:

* `onDiagnosisKeysRequested`
* `onExposure`

Basically, Android SDK only encapsulates contact matching. It doesn't have any say how users should be notified, how they would provide diagnosis keys to the server and how the list of diagnosis keys would be distributed to users.

**Comment by danbogdanov** [comment]
> It's true that there's a lot of overlap. Here are the methods you have to implement on top of Android API:
> 
> * `onDiagnosisKeysRequested`
> * `onExposure`

Understood, thank you! 
> Basically, Android Exposure Notification API only encapsulates contact matching. It doesn't have any say which users should be notified, how they would provide diagnosis keys to the server and how the list of diagnosis keys would be distributed to users.

Clear.

One thing I found hard to evaluate with my search skill set is the extra Bluetooth LE calibration work needed when one uses the Apple/Google solution. Are there any testing results on how well their exposure checking works, compared to DP-3T, Bluetrace, or others?

**Comment by Vanuan** [comment]
Define "well"

**Comment by danbogdanov** [comment]
well == detects contacts according to a well-established epidemiological standard/method that can be referenced or tuned.

I followed up on this with Srdjan and I got my responses. So we can close this.
